src/ucp/wireup/wireup.c

```diff
@@ -670,7 +670,7 @@ static ucs_status_t ucp_wireup_msg_handler(void *arg, void *data,
 }
 
 void ucp_wireup_assign_lane(ucp_ep_h ep, ucp_lane_index_t lane, uct_ep_h uct_ep,
-                            const char *info)
+                            int is_wireup_ep_connected, const char *info)
```

Comment:
instead of using this function, call ucp_wireup_ep_set_next_ep() directly for cm case

Reply:
This change goes from https://github.com/openucx/ucx/pull/4417, will fix there then resolve conflict here.